apps/api/src/modules/summary/entities/summary.entity.ts

```diff
@@ -2,13 +2,12 @@ import { ObjectType, Field } from '@nestjs/graphql';
 import { Entity, Column, PrimaryGeneratedColumn, UpdateDateColumn, CreateDateColumn } from 'typeorm';
 import { IsEnum, IsOptional } from 'class-validator';
 import { JobStatus, Status } from 'src/common/constants/summary';
-
-@Entity({ name: 'jobs' })
+@Entity({ name: 'summary' })
 @ObjectType()
 export class Summary {
   @PrimaryGeneratedColumn()
   @Field()
-  id: number;
+  job_id: number;
```

Comment:
This is incorrect usage. We follow camelCase.
if you want to map it with actual schema you use name property in Column decorator(PrimaryGeneratedColumn) here

Reply:
This is done